lib/debug/thread_client.rb

```diff
@@ -406,9 +410,9 @@ def show_src(frame_index: @current_frame_index,
           unless start_line
             if frame.show_line
               if dir > 0
-                start_line = frame.show_line
+                start_line = frame.show_line.lineno
```

Comment:
`frame.show_line` should already be an integer so this change is not need.

Reply:
Yeah, something was wrong with my environment. Removed.